statsmodels/genmod/generalized_estimating_equations.py

```diff
@@ -1656,7 +1656,7 @@ def _derivative_exog(self, params, exog=None, transform='dydx',
                                          self, params)
         return margeff
 
-    def qic(self, params, scale, cov_params):
+    def qic(self, params, scale, cov_params, nstep=1000):
```

Comment:
did you intentionally drop nstep by factor of 10 here?

I have a small preference for n_step.
I usually add the underline except for the old nobs.

Reply:
Thanks for the quick review.  I have changed nstep to n_step.  Using 10,000 steps as default is excessive and very slow, so I changed it to 1000.